src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Tests/Functions/ProcessCompletionOfNextDataSetVersionImportFunctionTests.cs

```diff
@@ -184,7 +93,7 @@ public async Task Success_PathNotUpdated()
 
         private async Task UpdateFileStoragePath(Guid instanceId)
         {
-            var function = GetRequiredService<ProcessCompletionOfNextDataSetVersionFunction>();
+            var function = GetRequiredService<ProcessCompletionOfNextDataSetVersionFunctions>();
```

Comment:
This file and the abstract class should probably be renamed to `ProcessCompletionOfNextDataSetVersionFunctions.cs` / `ProcessCompletionOfNextDataSetVersionFunctions`.

Reply:
Done thanks